frontend/src/lib/components/PropertyFilters/PropertyValue.js

```diff
@@ -1,66 +1,57 @@
 import React, { Component } from 'react'
 import api from '../../api'
-import AsyncCreatableSelect from 'react-select/async-creatable/dist/react-select.esm'
-import { selectStyle, debounce } from '../../utils'
+import { Select } from 'antd'
+import { debounce } from '../../utils'
 import PropTypes from 'prop-types'
 
 export class PropertyValue extends Component {
     constructor(props) {
         super(props)
-        this.state = { input: props.value }
+        this.state = {
+            input: '',
+            optionsCache: [],
+            options: [],
+        }
         this.loadPropertyValues = debounce(this.loadPropertyValues.bind(this), 250)
-        this.ref = React.createRef()
+        this.loadPropertyValues('')
     }
-    loadPropertyValues(value, callback) {
+    loadPropertyValues(value) {
         let key = this.props.propertyKey.split('__')[0]
+        this.setState({ optionsCache: { ...this.state.optionsCache, [value]: true } })
         api.get('api/' + this.props.endpoint + '/values/?key=' + key + (value ? '&value=' + value : '')).then(
```

Comment:
I think the no load on the values for personproperties is because the endpoint here needs to be matched according to the properties (event or person) that you want